### PR TITLE
Soaps nerf

### PR DESCRIPTION
**Pull request by eplgr**
Before, soaps had NO cooldown/took no time to clean things. This made them extremely godly compared to mops and any other thing that cleans floors/objects.

# In-game changelog
:cl: 
 * tweak: Soaps now take time to clean things

### CODE REVIEW

**Comment by Kurfursten** [comment]
If soap is gonna be garbage it should be cheaper at the biogenerator, 250 points is expensive for something worse than a mop

To make matters worse, iirc mops clean everything on the tile while soap has to be used once for every gib (although if you know how many gibs there are you can chain up several clean actions and run them simultaneously)

**Comment by eplgr** [comment]
@Kurfursten 
1) Do people really use the biogenerator?
2) If they do, do they buy soaps to clean things or to slip people? The slip feature obviously wasn't changed so if they buy it to slip people, it sounds like the price shouldn't change.
3) I don't think these topics have anything to do with my PR.

**Comment by Kurfursten** [comment]
Yes, they do to clean up the gibs from running chicken farming which produces fucktons of blood.

Good botanists don't ignore the biogenerator, it has a lot of useful stuff.

**Comment by eplgr** [comment]
The biogenerator truly sucks, imo. And it sounds like the only reason botanist use it is because soap was super imbalanced.

**Comment by Kurfursten** [comment]
It sounds like you're agreeing with me. Soap was tuned to an expensive price because it was good, therefore now that it will be bad it should be tuned in price to that.

(There are a lot of other great goodies at the biogenerator but I'm not here to extol the virtues of good botany, just talk balance.)

**Comment by jakmak6** [comment]
No need to nerf soap. What it allows is for people to easily clean up vomit/blood stains/oil stains that look beyond hideous. Not to mention since no one (excluding jani) really cares about cleaning no one really bothers. also cleaning up mass bloodstains help reduce lag since everytime you walk near blood it causes virus code to start running to see if you have transmittable diseases

**Comment by SkowronX** [comment]
drop this pr. Make mops instantly clean ~3x3 areas when wet (make it clean every decal on that tile). Soap only takes a single decal off each click.

**Comment by eplgr** [comment]
No, I believe cleaning should take time, and it makes no sense that the main tool a janitor has is far worse than a fukken soap. Actually, infinitely worse.

**Comment by carshalash** [comment]
Im pretty sure you know exactly why people use the mop.

**Comment by Lokiamis** [comment]
Nerfing soap isn't going to make janitors use the mop any more than they already do, and it certainly won't make them consider it their "main tool" so long as space cleaner and the janicart exist. Stations have become more populated since the mop was introduced and we just readded proper blood trails. 

If you want to see hot mop action there's only one solution: mini cleaning closets in maint for assistants and clowns to plunder as God intended (real suggestion, give assistants more to do to keep them productive)

**Comment by jknpj** [comment]
At least leave the syndi soap untouched, as it is supposed to quickly clean bloody messes

**Comment by eplgr** [comment]
The nerf to soaps has to do with it currently being in a completely broken state, that's it. It's got to be fixed.

**Comment by carshalash** [comment]
If you just want to see more people get slipped in hallways from mops then just say it man.

**Comment by Megamaw** [comment]
as a person who almost exclusively plays janitor

soap has its own niche. when you can't reach a thing with your upgraded janicart and the mess isn't several tiles big so there's no reason to waste your cleaner

taking that away, we'll just default to the cart + the cleaner.

instead of taking things away, think about the mop vs. the soap and try and come up with an appropriate niche for each, instead of just blindly nerfing soap because 'janitors aren't using their mops!'

a soap nerf this dumb will a: make janitors reconsider using their soap, because it's far weaker than it used to be, b: they're not going to use the mop anyways in its current state and c: even then, the soap is still better than the mop post-nerf.


The mop used to be good before messtrails. Not so much with the sprawling messes of oil and blood we have now.

tl;dr don't nerf the soap, find an inventive new way to buff the mop.

**Comment by eplgr** [comment]
One-click-make-dirt-disappear item is not balanced **no matter what**, it literally has no downside, and I'm 100% sure this was not intended.
Be objective, people, this may hurt your gameplay but you know soaps are currently not okay.

**Comment by Megamaw** [comment]
and you think this fixes the problem of mops being not used at all how?

people will just likely not use the soap or the mop anymore. are you going to nerf space cleaner next, because it's objectively better than the soap in most situations? (ie, multiple tiles in one click, all messes from three affected tiles cleaned instantly, has range unlike the soap)

or are you going to nerf the upgraded cart, for cleaning anything WITHOUT clicking? (and not using any resources)

the soap is a lot less busted than you think it is. do you even play as a fucking janitor? out of all the tools they have they actually use, it's their most niche tool. anyone who tells you otherwise is trying to sell you something.

soap, most likely.

**Comment by Lokiamis** [comment]
If you have such a strong opinion about soap it would be good if you made your stance on other cleaning methods clear. Are you setting is up for.a slippery slope of janicart and janiborg Holocausts if this gets merged? Even a bottle of space cleaner fits the "One-click-make-dirt-disappear" bill.

**Comment by Megamaw** [comment]
so let me understand the gameplan then

>nerf everything until everything is worse than mop

>mess powercreep outdoes the mop's cleaning power, leaving a perpetually messy station that gets nearly impossible to clean up unless there's 7 janitors

**Comment by Kurfursten** [comment]
The mop's gimmick is that it combines the reach of soap (able to hit walls) with the chemical transmission of a spray bottle. The "problem" is that the Janicart and Soap already do what Space Cleaner does.

On a **purely hypothetical level**, if you wanted people to use the mop, you should make chemistry matter more to janitors. Require alkaline cleaners for organic cleaning (e.g.: bleach/ammonia for gibs, blood, pie residue) and acidic cleaners for inorganic cleaning (e.g.: vinegar/sulfuric acid for robogibs, dirt, carbon from fires). Change the janicart upgrade to automatically pick up trash items to your specifications (a final solution to clowncart bananapocalypse?) rather than clean so it's not competing anymore.  Soap in this hypothetical world would simply enable anything to be cleaned by water by making it soluble (this is what soap does in real life). So, you can still clean up messes with water if chemistry is being a dick, but you have to carry around soap and pre-scrub before you wash.

On a **realistic, practical level** from someone who enjoys janitor now and then, there's nothing more satisfying than cleaning up a room filled with blood or carbon residue by gliding over it in your janicart. So you know, from a player perspective I can't let that go.

**Comment by Megamaw** [comment]
if any cleaning thing 'needs' to be nerfed, it's the cart's thing. i've proposed needing to do maintenance work on it with lube in the past.

ultimately as much as I want to see the mop see use again, unless someone has a real good idea i dont see it happenin

**Comment by ghost** [comment]
Janitors don't use the mop to clean, they use it to slip people and be dick head while cleaning. Why do you think they don't always get the janicart upgrade besides being new?

**Comment by AutismSpider** [comment]
is it so hard make soap have X number of uses? and add cargo crate that has random color soap to it?

**Comment by Dizor** [comment]
As Autism spider said, limit the uses, make it get dirty so it has to be cleaned, make it need to be combined with water to make soapy water. Its honestly just too easy to slide in, spam a few tiles and be done.

Janicart should be switched so that default its as is, when upgraded it dispenses the fluid put in it, so space cleaner cleans, anything else gets you lynched basically. If its dry it only should either A: Collect trash into the attached trash bags or B: simply drag the trash along until a release is used. Maybe even have it need to be recharged/batteries swapped. Using lube for maintenance is cool too but with how abusable the stuff is no chemist would give it to you regularly.

Thirdly, maybe add HE lights to upgraded auto/protolathe?

**Comment by Dizor** [comment]
Adding to that with the soap changes, see if  you cant add a "sudzy" state when it gets wet, makes it more slippery, and cleans better but gets used faster.

**Comment by cowboybuttsdrivemenuts** [comment]
Why tho?

**Comment by RoyalSev** [comment]
It's soap.

**Comment by Chazach** [comment]
man with significant janitor experience here,
Right now, the soap is for cleaning and the mop is for slipps. After the nerf, the soap will be for cleaning, and the mop will be for slipps, except cleaning will be slower and more painful. There is literally no reason to use the mop to  clean when spacecleaner exists, and no reason to use spacecleaner or soap when upgraded janicart exists. All that nerfing the soap will do without overhauling the rest of the jani kit is just make roundstart upgraded cart more of a necessity, and allow janiborg to completely eclipse their human counterpart in cleaning ability.

It's just soap, why is soap cleaning too fast such a disastrous issue? Janitor is such a non-vital role that people play just for fun or when they are new, forcing them to ditch soap is just strange. You say it makes no sense soap is better than a mop, but this PR makes no attempt to rebalance the mop to make it viable, nor does it seek to change the behaviour of the soap to make is less effective but still viable or fun to use; nerfing the soap purely by increasing its delay is a garbage solution to a non-issue.

**Comment by Blithering** [comment]
Soap is fine, buff mop. Until the ultra clean station objective is possible any call for a cleaning nerf is clearly a cultist plot.

**Comment by nuklearcellphoneg** [comment]
Oh man, my sides.

I've been nervous about posting my issues in the past because I was nervous about how they would be received since I knew it would irk the powergaming crowd, but I never once considered how touchy the issue of powercleaning was.

**Comment by Boxe** [comment]
is this really considered balance

**Comment by kernikov** [comment]
This PR is stupid and should be closed.

**Comment by Arthurdentist** [comment]
this isn't the first time this has been attempted and I doubt it will be the last.

Its such a trivial thing to nerf though. Is the spess station 13 experience damaged particularly because janitors can power clean? I don't know why we need to balance absolutely everything with delays and other pointless things.

**Comment by PJB3005** [comment]
Let's just end this before it's the next no fruit.

If you want soap to be less useful, buff the alternatives.

**Comment by eplgr** [comment]
@nuklearcellphoneg Powergamers will not divide us. We can beat them.
👺

**Comment by nuklearcellphoneg** [comment]
tbh i dont care about it i just think it's funny

**Comment by kilozombie** [comment]
I find it hilarious that eplgr thinks the biogenerator is never used
it's literally the best and most efficient source for meat, milk, and cheese

**Comment by Exxion** [comment]
God this would have made soap so shitty
Worse than a fucking wet rag cut out of a bedsheet with a shard

**Comment by Intigracy** [comment]
We could do the 3x3 mop suggestion as an upgraded mop.

**Comment by Probe1** [comment]
Why not just the default mop?

**Comment by Megamaw** [comment]
@nuklearcellphoneg i dont care about guns, tbh, only cleaning

**Comment by Shadowmech88** [comment]
What about a gun that cleans?

**Comment by Kurfursten** [comment]
Every bullet you load into the chamber polls for ghosts

When you fire, it dispenses a fully-clothed janitor equipped with a bar of soap
